euphonic/cli/powder_map.py

```diff
@@ -44,6 +44,11 @@ def get_parser() -> 'argparse.ArgumentParser':
                                help="Minimum |q| in 1/LENGTH_UNIT")
     sections['q'].add_argument('--q-max', type=float, default=3., dest='q_max',
                                help="Maximum |q| in 1/LENGTH_UNIT")
+    sections['plotting'].add_argument('--no-widgets', action='store_false',
+                                      dest='use_widgets', default=True,
```

Comment:
Confusingly, the `-h` help message for this argument says "default: True". It might be clearer to name the argument "no_widgets" and store_true?

Reply:
Good point, fixed.